scripts/check-broken-links.mjs

```diff
@@ -73,42 +114,78 @@ async function checkBrokenLinks() {
 
       linksChecked.push(result)
 
-      if (result.state === 'BROKEN') {
+      // 403 errors returned to the crawler by and large work for web users.
+      if (result.state === 'BROKEN' && result.status !== 403) {
         brokenLinks.push(result)
       }
     })
+    .on('retry', (retryDetails) => {
+      console.log(`Retrying ${retryDetails.url} ; status: ${retryDetails.status}`)
```

Comment:
This isn't strictly necessary but leaving it here to provide some visibility into how retries are going for a particular run.